src/activity_handlers.cpp

```diff
@@ -112,27 +112,34 @@ const std::map< activity_id, std::function<void( player_activity *, player *)> >
     { activity_id( "ACT_CHOP_TREE" ), chop_tree_finish }
 };
 
+void messages_in_process( player_activity *act, player *p ) {
```

Comment:
As you're making a new function, please change those pointers to (const) references. There is no need for them to be pointers.